pynsee/utils/save_df.py

```diff
@@ -87,22 +88,21 @@ def wrapper(*args, **kwargs):
                 _save_dataframe(df, file_name, parquet, silent)
             else:
                 try:
-                    if parquet:
-                        df = pd.read_parquet(file_name)
-                    else:
+                    try:
+                        mod = gpd if issubclass(obj, gpd.GeoDataFrame) else pd
```

Comment:
Wouldn't it be better yet to use isinstance? (In case we (ever) try to save a native geodataframe in the future)